bin/companion.sh

```diff
@@ -1,12 +1,8 @@
 #!/usr/bin/env bash
 
 # Load local env vars. In CI, these are injected.
-if [ -f .env.local ]; then
-  # https://gist.github.com/mihow/9c7f559807069a03e302605691f85572?permalink_comment_id=3625310#gistcomment-3625310
-  set -a
-  source <(sed -e '/^#/d;/^\s*$/d' -e "s/'/'\\\''/g" -e "s/=\(.*\)/='\1'/g" .env.local)
-  set +a
-  nodemon --watch packages/@uppy/companion/src --exec node ./packages/@uppy/companion/src/standalone/start-server.js
+if [ -f .env ]; then
+  nodemon --watch packages/@uppy/companion/src --exec node -r dotenv/config ./packages/@uppy/companion/src/standalone/start-server.js
```

Comment:
nice trick. did you test this locally with npm run start:companion?

Reply:
Yes looks like it's working